src/network/core/host.cpp

```diff
@@ -102,7 +102,18 @@ static void NetworkFindBroadcastIPsInternal(NetworkAddressList *broadcast) // GE
 		if (ifa->ifa_broadaddr->sa_family != AF_INET) continue;
 
 		NetworkAddress addr(ifa->ifa_broadaddr, sizeof(sockaddr));
-		if (!broadcast->Contains(addr)) *broadcast->Append() = addr;
+		//if (!broadcast->Contains(addr)) *broadcast->Append() = addr;
```

Comment:
I feel that this was by accident?

Reply:
The removal was intentional, lines 108-116 replace it, but I shouldn't have left it commented out.
TBH the following would be much better than a hand rolled loop.

``` c++
if (broadcast->end() == std::find(broadcast->begin(), broadcast->end(), addr)
    broadcast->push_back(addr);
```